src/main/java/FindTaskCommand.java

```diff
@@ -21,6 +31,11 @@ public void execute(TaskList tasks, Ui ui, Storage storage) throws IOException,
         }
     }
 
+    /**
+     * Returns a boolean value signalling whether the program should exit.
+     *
+     * @return A boolean value indicating whether the program should exit.
+     */
```

Comment:
Can be clear of which boolean value you are returning, eg. (true/false) or (1/0).